packages/p2p-v2/src/pages/buy-sell/components/CurrencyDropdown/CurrencyDropdown.scss

```diff
@@ -0,0 +1,64 @@
+.p2p-v2-currency-dropdown {
+    &__dropdown {
+        display: inline-flex;
+        align-items: center;
+        justify-content: space-between;
+        position: relative;
+        border-radius: 4px;
+        text-align: left;
+        padding: 7px 8px;
+        border: 1px solid #d6dadb;
+        cursor: pointer;
+        width: 9rem;
+
+        @include mobile {
+            padding: 3px 8px;
+        }
+
+        &:hover:not(.p2p-v2-currency-dropdown__dropdown--active) {
+            border-color: #999;
+        }
+
+        &--active {
+            border-color: #85acb0;
+        }
+
+        &-icon {
+            transition: transform 0.2s cubic-bezier(0.25, 0.1, 0.25, 1);
+
+            &--active {
+                transform: rotate(180deg);
+            }
+        }
+
+        &-text {
+            font-size: 10px;
```

Comment:
why specify the font size here?

Reply:
there wasn't a smaller font-size for this 😅 so i had to do 10px

Reply:
dont we have 2xs?

Reply:
ah yeah they have, will amend this 👍